### ISSUE TITLE
Maybe wmi need to be added to the windows requirement?

### ISSUE THREAD

**Issue by dglaude** [Neradoc/discotool#3]
Not sure exactly how the requirements-win32.txt file work, but I believe I followed the instruction and wmi was missing.
So I had to install it with pip and now it works for me.

Regards

> C:\Users\MediaMonster>C:\Users\MediaMonster\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\Scripts\discotool.exe
> Traceback (most recent call last):
>   File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.8_3.8.2800.0_x64__qbz5n2kfra8p0\lib\runpy.py", line 194, in _run_module_as_main
>     return _run_code(code, main_globals, None,
>   File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.8_3.8.2800.0_x64__qbz5n2kfra8p0\lib\runpy.py", line 87, in _run_code
>     exec(code, run_globals)
>   File "C:\Users\MediaMonster\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\Scripts\discotool.exe\__main__.py", line 4, in <module>
>   File "C:\Users\MediaMonster\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\site-packages\discotool\__init__.py", line 1, in <module>
>     from .usbinfos import get_devices_list
>   File "C:\Users\MediaMonster\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\site-packages\discotool\usbinfos\__init__.py", line 25, in <module>
>     from .usbinfos_win32 import get_devices_list
>   File "C:\Users\MediaMonster\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\site-packages\discotool\usbinfos\usbinfos_win32.py", line 5, in <module>
>     import wmi
> ModuleNotFoundError: No module named 'wmi'
> 
> C:\Users\MediaMonster>python3 -m pip install wmi
> Collecting wmi
>   Downloading WMI-1.5.1-py2.py3-none-any.whl (28 kB)
> Collecting pywin32
>   Downloading pywin32-301-cp38-cp38-win_amd64.whl (9.3 MB)
>      |████████████████████████████████| 9.3 MB 6.4 MB/s
> Installing collected packages: pywin32, wmi
> Successfully installed pywin32-301 wmi-1.5.1
> 
> C:\Users\MediaMonster>C:\Users\MediaMonster\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\Scripts\discotool.exe
> - Périphérique Série Usb (Com17)
>         Microsoft [SN:7F7226641F9A4030]
>         COM17 (REPL)
>         D:\
>

**Comment by dglaude** [comment]
Did I mention this was on Windows 10?

**Comment by Neradoc** [comment]
Ah yes thanks, I'm in the process of figuring out how to correctly specify the OS specific requirements for the pypi install.
I though wmi was installed with pywin32, but it's the other way around, I should put both anyway !

For manual install, the way requirements files work is you normally do:
```py
pip install -r requirements-win32.txt
```

**Comment by Neradoc** [comment]
I believe installing is fixed in `0.2.5`.
I tested doing:
```powershell
pip uninstall discotool-for-microcontrollers pywin32 wmi
pip install -U discotool-for-microcontrollers
py -m discotool
```
